qiskit/aqua/components/uncertainty_models/multivariate_variational_distribution.py

```diff
@@ -49,26 +48,47 @@ def __init__(self,
             high = np.ones(len(num_qubits))
         self._num_qubits = num_qubits
         self._var_form = var_form
+        # fix the order of the parameters in the circuit
+        if isinstance(self._var_form, QuantumCircuit):
+            self._var_form_params = list(self._var_form.parameters)
+        else:
+            warnings.warn('The VariationalForm type is deprecated as argument of the '
+                          'UnivariateVariationalDistribution as of 0.7.0 and will be removed no '
```

Comment:
This has the wrong name it should me Multivariate.....

Reply:
Oh, of course! Good catch 👍